core/transport/src/grpc/file_system_master.proto

```diff
@@ -145,6 +146,7 @@ message GetStatusPOptions {
   optional LoadMetadataPType loadMetadataType = 1;
   optional FileSystemMasterCommonPOptions commonOptions = 2;
   optional Bits accessMode = 3;
+  optional bool updateTimestamps = 4 [default = true];
```

Comment:
why is this called `updateTimestamps` and the other is called `updateLastAccessTime`? Also, which timestamps is this referring to?

Reply:
`OpenFilePOptions` is only for reading files and only last access time is relevant; `GetStatusPOptions` is used in `getStatus` call which is called for read, write and other operations, so it can be other mutable timestamps as well. It is not specific to a certain timestamp type but more to indicate that the call is for an internal operation that does not imply any user interactions.